homeassistant/components/media_player/webostv.py

```diff
@@ -87,6 +87,7 @@ def setup_tv(host, mac, name, customize, config, hass, add_devices):
     from pylgtv import WebOsClient
     from pylgtv import PyLGTVPairException
     from websockets.exceptions import ConnectionClosed
+    from concurrent import futures
```

Comment:
Instead of this, `import asyncio` at the top of the module, and use `asyncio.TimeoutError`. It's an alias for `concurrent.futures.TimeoutError`.